q2_fondue/entrezpy_clients/_elink.py

```diff
@@ -1,82 +0,0 @@
-# ----------------------------------------------------------------------------
```

Comment:
We don't need that. It's better to use the implementation found in `entrezpy` as it can handle linking different requests better.